src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -152,7 +153,17 @@ const BrazeBannerWithSatisfiedDependencies = ({
     meta,
 }: InnerProps) => {
     useEffect(() => {
+        // Log the impression with Braze
         meta.logImpression();
+
+        // Log VIEW event with Ophan
+        submitComponentEvent({
+            component: {
+                componentType: 'BRAZE_BANNER',
+                id: meta.dataFromBraze.componentName,
+            },
+            action: 'VIEW',
+        });
```

Comment:
According to the types, this is the minimum data we need to provide to `submitComponentOphanEvent`. Is there other data we should be sending?

Reply:
> According to the types, this is the minimum data we need to provide to `submitComponentOphanEvent`. Is there other data we should be sending?

Just to be clear, you're mentioning `submitComponentOphanEvent` here but using `submitComponentEvent` above:

https://github.com/guardian/dotcom-rendering/blob/adf3a321e24a369ed496e6bfd83da212a0816694/src/web/components/StickyBottomBanner/BrazeBanner.tsx#L160

..and they are two different functions (tho `submitComponentOphanEvent` calls `submitComponentEvent`):

https://github.com/guardian/dotcom-rendering/blob/ab86bc6f16d28ad703465e635af1dedb242f124a/src/web/browser/ophan/ophan.ts#L99-L132

Anyway - I _think_ what you have is indeed enough information - you can check out [`extractors.ComponentEventExtractor.ComponentRepr`](https://github.com/guardian/ophan/blob/de959dbe4b506d97e56181c0688d0e6d94446744/the-slab/app/extractors/ComponentEventExtractor.scala#L14-L20) in Ophan for the definitive word on what you can send, with some examples in the [`ComponentEventExtractorTest`](https://github.com/guardian/ophan/blob/de959dbe4b506d97e56181c0688d0e6d94446744/the-slab/test/extractors/ComponentEventExtractorTest.scala).

Reply:
Thanks @rtyley that's really useful, I'll check out the links. Ah sorry yeah that's a mistake on my end (the names are v similar!), I meant `submitComponentEvent`. Like you said, `submitOphanComponentEvent` calls `submitComponentEvent` under the hood. It looks like the former is there to make the call more convenient (presumably it's common to send the params expected by `sendOphanComponentEvent`?)